living_documentation_generator/generator.py

```diff
@@ -223,34 +327,40 @@ def _generate_md_issue_page(self, issue_page_template: str, consolidated_issue:
             "date": date,
             "page_issue_title": title,
             "issue_summary_table": issue_table,
-            "issue_content": issue_content
+            "issue_content": issue_content,
         }
 
         # Run through all replacements and update template keys with adequate content
         issue_md_page_content = issue_page_template.format(**replacements)
 
         # Save the single issue Markdown page
         page_filename = consolidated_issue.generate_page_filename()
-        with open(os.path.join(self.output_path, page_filename), 'w', encoding='utf-8') as f:
+        with open(os.path.join(self.output_path, page_filename), "w", encoding="utf-8") as f:
             f.write(issue_md_page_content)
 
         logger.debug("Generated Markdown page: %s.", page_filename)
 
-    def _generate_index_page(self, issue_index_page_template: str, consolidated_issues: dict[str, ConsolidatedIssue])\
-            -> None:
+    def _generate_index_page(
+            self,
+            issue_index_page_template: str,
+            consolidated_issues: dict[str, ConsolidatedIssue],
+    ) -> None:
         """
         Generates an index page with a summary of all issues and save it to the output directory.
 
-        :param issue_index_page_template: The template string for generating the index page.
-        :param consolidated_issues: The dictionary containing all issues data.
+        Args:
+            issue_index_page_template (str): The template string for generating the index markdown page.
+            consolidated_issues (dict): A dictionary containing all consolidated issues.
         """
 
         # Initializing the issue table header based on the project mining state
         if self.__project_state_mining_enabled:
-            issue_table = ("| Organization name | Repository name | Issue 'Number - Title' |"
-                           " Linked to project | Project status | Issue URL |\n"
-                           "|-------------------|-----------------|------------------------|"
-                           "-------------------|----------------|-----------|\n")
+            issue_table = (
```

Comment:
I'd consider extracting this to some constants.py or at least on top of the file as global constant perhaps - it's good to have visibility over these things

Reply:
Issue tables for index page are now moved into constants.py file ([5db1778](https://github.com/AbsaOSS/living-doc-generator/pull/30/commits/5db1778f2ea5b37dd99c4b062d94c7c3ed9eded2)).